Source/WebKit/UIProcess/PDF/WKPDFHUDView.mm

```diff
@@ -254,13 +254,13 @@ - (void)_performActionForControl:(NSString *)control
     if (!page)
         return;
     if ([control isEqualToString:PDFHUDZoomInControl])
-        page->pdfZoomIn(_pluginIdentifier);
+        page->pdfZoomIn(*_pluginIdentifier);
```

Comment:
How do we know `!!_pluginIdentifier`?

Reply:
Because it is set unconditionally in `initWithFrame`.  With Objective C, I didn't find a way to avoid using Markable even though this data member is always initialized in the "constructor".